src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -27,14 +27,24 @@
 
     public static final String VALID_NAME_AMY = "Amy Bee";
     public static final String VALID_NAME_BOB = "Bob Choo";
+    public static final String VALID_NAME_CEDAR = "Cedar Mush";
+    public static final String VALID_NAME_DELON = "Delon Fattie";
     public static final String VALID_PHONE_AMY = "11111111";
     public static final String VALID_PHONE_BOB = "22222222";
+    public static final String VALID_PHONE_CEDAR = "33333333";
+    public static final String VALID_PHONE_DELON = "444444444";
     public static final String VALID_EMAIL_AMY = "amy@example.com";
     public static final String VALID_EMAIL_BOB = "bob@example.com";
+    public static final String VALID_EMAIL_CEDAR = "cedar@example.com";
+    public static final String VALID_EMAIL_DELON = "delon@example.com";
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
+    public static final String VALID_ADDRESS_CEDAR = "Block 123, Cedary Street 3";
+    public static final String VALID_ADDRESS_DELON = "Block 123, Dragon Street 3";
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_TAG_MUSHROOM = "mushroom";
+    public static final String VALID_TAG_FATTIE = "fattie";
 
     public static final String NAME_DESC_AMY = " " + PREFIX_NAME + VALID_NAME_AMY;
```

Comment:
Very curious about why you added `CEDAR` and `DELON` into this part and `TypicalPersons` below since you did not seem to have utilised them for any testing at all in the end, so actually what was already given would have been sufficient...?